### PR TITLE
Sandbox/add GitHub email action

### PR DESCRIPTION
**Pull request by mdkinney**

### CODE REVIEW

File: .github/workflows/code-review.yml [hunk 1]
```diff
@@ -31,9 +31,8 @@ jobs:
         architecture: 'x64'
     - name: Gen mail
       run: |
-        echo 'Subject: PR #${{ github.event.pull_request.number }} Updated SHA ${{ github.event.pull_request.head.sha }}' > mail.txt
```
Comment (mdkinney) [inline .github/workflows/code-review.yml]: This format looks good

File: .github/workflows/code-review.yml [hunk 2]
```diff
@@ -31,9 +31,8 @@ jobs:
         architecture: 'x64'
     - name: Gen mail
       run: |
-        echo 'Subject: PR #${{ github.event.pull_request.number }} Updated SHA ${{ github.event.pull_request.head.sha }}' > mail.txt
-        echo.>> mail.txt
-        echo 'PR Notification' >> mail.txt
+        echo -e "Subject: PR #${{ github.event.pull_request.number }} Updated SHA ${{ github.event.pull_request.head.sha }}\n" > mail.txt
```
Comment (mdkinney) [inline .github/workflows/code-review.yml:35]: This is even better

File: .github/workflows/code-review.yml [hunk 3]
```diff
@@ -31,9 +31,10 @@ jobs:
...
     - name: Gen mail
       run: |
-        echo Subject: PR #${{ github.event.pull_request.number }} Updated SHA ${{ github.event.pull_request.head.sha }} > mail.txt
-        echo . >> mail.txt
-        echo PR Notification >> mail.txt
+        echo 'Subject: PR #${{ github.event.pull_request.number }} Updated SHA ${{ github.event.pull_request.head.sha }}' > mail.txt
+        echo.>> mail.txt
+        echo 'PR Notification' >> mail.txt
+        more mail.txt
     - name: Send mail
```
Comment (mdkinney) [inline .github/workflows/code-review.yml:38]: Using a single quote does not work.  Need to use double quotes.

File: .github/workflows/code-review.yml (hunk 1 above)
Comment (mdkinney) [reply .github/workflows/code-review.yml]: This is a reply
That spans **multiple** lines
And uses some markdown

**Review by mdkinney — COMMENTED** [review]
The subject line is missing dir/module reference

File: .github/workflows/code-review.yml (hunk 3 above)
Comment (mdkinney) [reply .github/workflows/code-review.yml:38]: All echo commands updated to use ""

**Review by svenur — APPROVED** [review]
Looks good to me, and also received the email notification.

**Pull request merged**